astropy/config/configuration.py

```diff
@@ -98,6 +98,10 @@ class Conf(_config.ConfigNamespace):
                 aliases=['astropy.utils.console.USE_COLOR'])
         conf = Conf()
     """
+    def __iter__(self):
+        for key, val in self.__class__.__dict__.items():
+            if isinstance(val, ConfigItem):
+                yield key
```

Comment:
Although is yielding just the key enough? Or you rather have it yield both key and value?

@saimn , since you recently worked with `config`, do you have an opinion?

Reply:
I'd definitely yield just the key, as that is how it works for `dict` too. We could add an `items()` if we ever wanted to have both.

Reply:
Yes, yielding keys is better. I also think it would be useful to have `.keys()`, `.items()`, etc.

Reply:
Well, I still have a few hours before deadline... 💭